app/react/components/Elements/FeatureToggle.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Immutable from 'immutable';
+
+export type ComponentPropTypes = {
+  featureActivated: boolean;
+  children: React.ReactNode;
+};
+
+export type OwnPropTypes = {
+  feature: string;
+};
+
+const FeatureToggle: React.FC<ComponentPropTypes> = ({
+  featureActivated,
+  children,
+}: ComponentPropTypes) => (featureActivated ? <React.Fragment>{children}</React.Fragment> : null);
```

Comment:
@RafaPolit i do not think a fragment is needed here ? just 
`featureActivated ? children : null`, or im missing something ?

Reply:
Wouldn't that fail if children are an array of elements instead of a single one?  I'll test.  Maybe I'm thinking about previous versions or I am completely mistaken.

Reply:
I cannot seem to get rid of the React.Fragment without having the TS fail on the assignment as return of the Component.  I could use some help.

Reply:
its ok, i thought only removing the fragment will work, but its not worth it to spend the time if that is not the case. its ok like this.